src/Winium.Cruciatus/Core/RotationManager.cs

```diff
@@ -0,0 +1,207 @@
+            [MarshalAs(UnmanagedType.U2)]
+            public UInt16 dmSize;
+
+            [MarshalAs(UnmanagedType.U2)]
+            public UInt16 dmDriverExtra;
+
+            [MarshalAs(UnmanagedType.U4)]
+            public UInt32 dmFields;
+
+            public POINTL dmPosition;
+
+            [MarshalAs(UnmanagedType.U4)]
+            public UInt32 dmDisplayOrientation;
+
+            [MarshalAs(UnmanagedType.U4)]
+            public UInt32 dmDisplayFixedOutput;
+
+            [MarshalAs(UnmanagedType.I2)]
+            public Int16 dmColor;
+
+            [MarshalAs(UnmanagedType.I2)]
+            public Int16 dmDuplex;
+
+            [MarshalAs(UnmanagedType.I2)]
+            public Int16 dmYResolution;
+
+            [MarshalAs(UnmanagedType.I2)]
+            public Int16 dmTTOption;
+
+            [MarshalAs(UnmanagedType.I2)]
+            public Int16 dmCollate;
+
+            [MarshalAs(UnmanagedType.ByValTStr, SizeConst = 32)]
+            public string dmFormName;
+
+            [MarshalAs(UnmanagedType.U2)]
+            public UInt16 dmLogPixels;
+
+            [MarshalAs(UnmanagedType.U4)]
+            public UInt32 dmBitsPerPel;
+
+            [MarshalAs(UnmanagedType.U4)]
+            public UInt32 dmPelsWidth;
+
+            [MarshalAs(UnmanagedType.U4)]
+            public UInt32 dmPelsHeight;
+
+            [MarshalAs(UnmanagedType.U4)]
+            public UInt32 dmDisplayFlags;
+
+            [MarshalAs(UnmanagedType.U4)]
+            public UInt32 dmDisplayFrequency;
+
+            [MarshalAs(UnmanagedType.U4)]
+            public UInt32 dmICMMethod;
+
+            [MarshalAs(UnmanagedType.U4)]
+            public UInt32 dmICMIntent;
+
+            [MarshalAs(UnmanagedType.U4)]
+            public UInt32 dmMediaType;
+
+            [MarshalAs(UnmanagedType.U4)]
+            public UInt32 dmDitherType;
+
+            [MarshalAs(UnmanagedType.U4)]
+            public UInt32 dmReserved1;
+
+            [MarshalAs(UnmanagedType.U4)]
+            public UInt32 dmReserved2;
+
+            [MarshalAs(UnmanagedType.U4)]
+            public UInt32 dmPanningWidth;
+
+            [MarshalAs(UnmanagedType.U4)]
+            public UInt32 dmPanningHeight;
+        }
+
+        #endregion
+
+        #region Methods
+
+        [DllImport("User32.dll")]
+        [return: MarshalAs(UnmanagedType.Bool)]
+        private static extern Boolean EnumDisplaySettings(
+            [param: MarshalAs(UnmanagedType.LPTStr)]
+            string lpszDeviceName,
+            [param: MarshalAs(UnmanagedType.U4)]
+            int iModeNum,
+            [In, Out]
+            ref DEVMODE lpDevMode);
+
+        [DllImport("User32.dll")]
+        [return: MarshalAs(UnmanagedType.I4)]
+        private static extern int ChangeDisplaySettings(
+            [In, Out]
+            ref DEVMODE lpDevMode,
+            [param: MarshalAs(UnmanagedType.U4)]
+            uint dwflags);
+
+
+        /// <summary>
+        /// Gets the current orientation of the screen
```

Comment:
This worked for primary screen, when using more one monitor. I think need change to `Gets the current orientation of the primary screen`